.vscode/settings.json

```diff
@@ -1,4 +1,8 @@
 {
-  "editor.tabSize": 2,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.codeActionsOnSave": {
```

Comment:
I'm not a huge fan of that, but maybe if it's limited to the repo and not messing with other people's projects I open it's enjoyable.